packages/ui/src/components/molecules/card/card-content.tsx

```diff
@@ -0,0 +1,10 @@
+import { cn } from "@repo/ui/utils";
+import { forwardRef } from "react";
+
+export const CardContent = forwardRef<
+  HTMLDivElement,
+  React.HTMLAttributes<HTMLDivElement>
+>(({ className, ...props }, ref) => (
+  <div ref={ref} className={cn("p-6", className)} {...props} />
+));
+CardContent.displayName = "CardContent";
```

Comment:
Not something for this PR, but should we be setting `displayName` for all components, even the existing ones in flame-defi?

Reply:
Yeah me neither! Well, I consistently left them off haha.